app/helpers/audited_helper.rb

```diff
@@ -0,0 +1,10 @@
+module AuditedHelper
+  def enumerize(class_of_item, property, value)
+    # If this is an enum and the value is found inside that enum, look it up in I18n by the class name, the property name, and the enum value:
```

Comment:
Maybe this would be easier to understand if the comment contained an example of input and output?